pkg/db/tracing.go

```diff
@@ -52,8 +52,8 @@ func (d traceableDB) ExecContext(ctx context.Context, query string, args ...inte
 	defer func() {
 		d.FinishSpan(span, err)
 	}()
-	span.SetTag("sql", query)
-	logrus.WithTime(time.Now()).WithField("sql.method", "ExecContext").Debug(query)
+	span.LogKV("sql", query)
```

Comment:
sql shouldn't be a tag, because it has a much shorter truncation in jaeger compared to logs, using logkv allows us to see  more of the sql for debugging